pkgs/applications/networking/mailreaders/himalaya/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, openssl, pkg-config, installShellFiles, enableCompletions ? stdenv.hostPlatform == stdenv.buildPlatform }:
+rustPlatform.buildRustPackage rec {
+  pname = "himalaya";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "soywod";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fl3lingb4wdh6bz4calzbibixg44wnnwi1qh0js1ijp8b6ll560";
+  };
+
+  cargoSha256 = "10p8di71w7hn36b1994wgk33fnj641lsp80zmccinlg5fiwyzncx";
+
+  nativeBuildInputs = [ openssl pkg-config installShellFiles ];
```

```suggestion
  nativeBuildInputs = [ installShellFiles ]
    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ openssl pkg-config ];
```

I haven't noticed it the first time around, but why do you have openssl in both `nativeBuildInputs` and `buildInputs`? Does it not work with just `buildInputs`?

---

Thanks a lot for picking bug this out. :)